guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -2929,12 +2931,11 @@
 01:45:14.128 --> 01:45:16.010
 [Lars Johansson]: It's fascinating to follow you.
 
-01:45:16.830 --> 01:45:20.954
+01:45:16.837 --> 01:45:19.490
 [Lars Johansson]: I thought I would ask a question about...
-[Stephen Frost]: Sorry, Lars, whose speech was that?
 
-01:45:21.034 --> 01:45:21.715
-[Stephen Fros]: I'm so sorry.
+01:45:19.515 --> 01:45:21.715
+[Stephen Frost]: Sorry, Lars, whose speech was that? I'm so sorry.

```

Comment:
I joined the lines here as it seemed to make sense based on the video.

Reply:
Good catch on the speaker's name.